README.md

```diff
@@ -79,7 +94,8 @@ Will run django tests using `--keepdb` and `--parallel`. You may pass a path to
 `make test someapp.tests.test_views`
 
 ### Adding new pypi libs
-Just run `pipenv install LIB_NAME_ON_PYPI` and then `pipenv lock` to lock the version in Pipfile.lock file
+Add the libname to either requirements.in or dev-requirents.in, then compile it and install.
+`pip-compile requirements.in > requirements.txt && pip install -r requirements.txt`
```

Comment:
how about adding something like this to instruct on how to update dependencies?
https://github.com/vintasoftware/django-zombodb/blob/master/Makefile#L54